src/core/Details/Details.md

```diff
@@ -0,0 +1,70 @@
+Examples:
+
+- [Basic use](./#/Components/Details?id=basic-use)
+- [Varied content](./#/Components/Details?id=varied-content)
+
+<div style="margin-bottom: 40px">
+  [Props & methods](./#/Components/Toast?id=props--methods)
```

Comment:
Wrong link to Toast component (copy paste error)

Reply:
Ah, good call. Should now be correct.